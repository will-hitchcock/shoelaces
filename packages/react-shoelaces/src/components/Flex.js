import styled from 'styled-components'
import { flex } from 'shoelaces'

export const Flex = styled.div`${flex}`

export default Flex
