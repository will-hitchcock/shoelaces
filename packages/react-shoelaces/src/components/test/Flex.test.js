import Flex from '../Flex'

describe('<Flex />', () => {
  it('should render a <Flex />', () => {
    expect(shallow(<Flex />)).toMatchSnapshot()
  })
})
