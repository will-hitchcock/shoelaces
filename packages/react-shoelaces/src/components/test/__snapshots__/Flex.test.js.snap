// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex /> should render a <Flex /> 1`] = `
<div
  className="sc-bdVaJa iqnCHL"
/>
`;
